subprojects/internal-testing/src/main/groovy/org/gradle/util/FlakyTest.groovy

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.util
```

Comment:
Let's not have this in the public API package `org.gradle.util`. Or maybe it doesn't matter?

Reply:
I remember we once clarified `org.gradle.util` is not a public package. But `org.gradle.test.fixtures` is a better location for this annotation.